examples/mock-filesystem/tests/directory-reader.spec.js

```diff
@@ -0,0 +1,64 @@
+// @ts-check
+const { test, expect } = require('@playwright/test');
+
+test.beforeEach(async ({page}) => {
+  await page.addInitScript(() => {
+    class FileSystemHandleMock {
+      constructor({name, children}) {
+        this.name = name;
+        children ??= [];
```

Comment:
Do all browsers understand that already? Or does it go through babel?

Reply:
dunno, tested with chrome only and the example doesn't run other browsers, we can change it to the old syntax

Reply:
works in all browsers